### DISCUSSION TITLE
doris内存持续增长

### DISCUSSION THREAD

**Discussion by modric-jrc** [apache/doris#9970] [A - General / Q&A]
我内网搭建了doris集群
fe: 192.168.1.5
be: 192.168.1.5(4C16G 1T磁盘)
      192.168.1.6(4C16G 1T磁盘)
      192.168.1.7(4C16G 1T磁盘)
启动后查看health：
{"status": "OK","msg": "To Be Added"}
然后将CK的数据从外网拉到内网写入csv文件，然后再讲csv文件通过 curl --location-trusted -u xx -H xxx -T xxx.csv http://192.168.50.53:8030/api/online/xxxx/_stream_load 的方式导入doris数据库。
然后导入的时候发现三个后端的be，有一台(192.168.1.7)内存会持续增长，从16%会一直增加到80以上，然后挂掉
请问下这是什么原因导致的？

**Comment by dataalive** [comment]
这个用的是什么版本？
